index.html

```diff
@@ -239,13 +238,11 @@ <h2 id="x2.link-type-preload">Link type "<code>preload</code>"</h2>
       link and a later fetch requesting the same resource.</p>
     </section>
     <section>
-      <h2>`link` element extensions</h2>
-      <p>[[!HTML]] defines the `as` content and IDL attributes. The state of
-      the `as` content attribute is taken into account in the above processing
-      model, which will set the <a>request destination</a> as well as the
-      corresponding <a>request type</a> and <a>request initiator</a>. This is necessary
-      to guarantee correct prioritization, request matching, application of the
-      correct [[!CSP3]] policy, and setting of the appropriate `Accept` request
+      <h2>`as` attribute</h2>
+      <p class=note>[[HTML]] [defines](https://html.spec.whatwg.org/#attr-link-as) the
+      `as` content and IDL attributes. The attribute is necessary to guarantee
+      correct prioritization, request matching, application of the correct
+      [[!CSP3]] policy, and setting of the appropriate `Accept` request
```

Comment:
Guessing, other way around.. We already have normative HTML ref, right? If so, keep it as such, but CSP3 should be informative?